### PR TITLE
Fix spacing to reveal multiple issues. 

### PR DESCRIPTION
**Pull request by bdougie**
The cause to the background not updating very simple. Once you fix all the spacing you can see you have a closed div nested in a `Link`. There is a warning in your console mentioning this, but it doesn't really explain what is going on. 

I urge to no wait, but download [Atom](https://atom.io/) or [VSCode](https://code.visualstudio.com/) now to prevent spending way too much time on something Xcode is not doing for you. 

I download a plugin called prettier to quickly fix spacing.

### CODE REVIEW

File: src/components/Library.js [hunk 1]
```diff
@@ -1,39 +1,35 @@
...
+  render() {
+    return (
+      <main style={{backgroundColor: "red"}} className="collection">
+        <section className="album-covers container clearfix">
+          {this.state.albums.map((album, index) => (
+            <div className="collection-album-container column fourth" key={index}>
+              <p>
+                <img src={album.albumCover} alt={album.title} />
+                <Link to={`/album/${album.slug}`}>
+                  <div className="collection-album-info caption" />
```
Comment (bdougie) [inline src/components/Library.js:21]: This is breaking everthing. Deleted the `Link` and all divs inside the the `<p>` on line 18

File: src/components/Library.js [hunk 2]
```diff
@@ -1,39 +1,35 @@
...
+                </Link>
+                <div className="album-name">{album.title}</div>
```
Comment (bdougie) [inline src/components/Library.js:23]: This div should not be here

**Comment by bdougie** [comment]
@JMena212 Your render method should look like this

```js
 render() {
    return (
      <div className="album-covers container clearfix">
        {this.state.albums.map((album, index) => (
          <div className="collection-album-container column fourth" key={index}>
            <img src={album.albumCover} alt={album.title} />
            <div className="collection-album-info caption">
              <p>
                <a href="#"className="album-name">{album.title}</a>
                <a href="#">{album.artists}</a>
                {album.songs.length} songs
              </p>
            </div>
          </div>
        ))}
      </div>
    );
  }

```
adding this to your index.css fixes mostly everything, after spacing and random divs are removed. Forget about my mention 

```css
body {
  background-image: url(/assets/images/blurred_backgrounds/blur_bg_3.jpg);
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-position: center center;
  background-size: cover;
  margin: 0;
  padding: 0;
  font-family: 'Open Sans', sans-serif;
  font-size: 18px;
}
```

Hope some of this helps

**Comment by JMena212** [comment]
Thanks, Brian. That was very helpful. I'll be downloading Atom or VSCode to spare me additional issues.
I did have to add the album.slug for the href to make the link function.

`<a href={`/album/${album.slug}`}className="album-name">{album.title}</a>`

**Pull request closed without merging**